### ISSUE TITLE
CTD when renaming a soldier with "é" on a french keyboard.

### ISSUE THREAD

**Issue by BeornTB** [OpenApoc/OpenApoc#471]
https://www.dropbox.com/s/27y16ussclln3kn/330-CTD-Renaming.zip?dl=0

Title.

Anomalies also encountered (as seen on Discord):
- No cityscape pop-ups for alien containment, daily score or weekly income
- No weekly stock renewal: flying suits, Hawk, etc show on the trade screens as unavailable/sold-out (grayed out)
- No weekly income
- No tooltip for soldier stats on the equip screen
- Seemingly no alien infiltration screen
- Soldiers had too many TUs to be true: is weight vs strength implemented?
- Everyone hates me (maybe they should? ;) )

Thanks again,
B

**Comment by FranciscoDA** [comment]
This should be fixed once #472 is merged.
I suggest that you gather more information about the other issues. i.e: observed vs expected behaviour, screenshots and logs, where appropriate. Then, create separate issues for each one so that we can solve and close them individually. Otherwise managing the issue tracker becomes difficult

Btw: I checked the code for stock renewal and it should be working, except that the item stock is updated on tuesdays (not mondays), so you may see the new equipment greyed out and the score screen will display "Week 2" since monday (even though the economy wasn't resupplied)

**Comment by BeornTB** [comment]
Great, that answers that. Skipped a day, inventory replenished. New issue though: almost all prices are doubled (200k instead of 100k for a Hawk, for instance). A very few (Hovercar for instance) remained the same. 

Also, I can't seem to take screenshots in-game as it only returns a loading screen (big X openapoc). I intended to put a few here indeed.

Gonna keep playing 'til assault ships to see if I can reproduce the invisible assault bug. And abuse stun raids cuz I ain't made of money :p
